### PR TITLE
add necessary docs for issue 48

### PR DESCRIPTION
**Pull request by humorless**
add necessary documentation for #48

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,6 +139,18 @@ runtime didn't start up properly (check your node process for instance), maybe
 the compiles CLJS caused an error (anything in the browser console)? Maybe it's
 a networking issue... We handed over control, and never got it back.
 
+- `"Execution error (ReferenceError) at (<cljs repl>:1).\ndocument is not defined\n"`
+
+This usually means you're using libraries that expect a DOM (i.e. they expect to
```
Comment (alysbrooks) [inline README.md:144]:
```suggestion
This usually means you're using libraries that expect a DOM (i.e., they expect to 
```

File: README.md [hunk 2]
```diff
@@ -139,6 +139,18 @@ runtime didn't start up properly (check your node process for instance), maybe
 the compiles CLJS caused an error (anything in the browser console)? Maybe it's
 a networking issue... We handed over control, and never got it back.
 
+- `"Execution error (ReferenceError) at (<cljs repl>:1).\ndocument is not defined\n"`
+
+This usually means you're using libraries that expect a DOM (i.e. they expect to 
+find`js/document`), but you are using them in an environment where there is no DOM,
```
Comment (alysbrooks) [inline README.md:145]:
```suggestion
find `js/document`), but you are using them in an environment where there is no DOM,
```

File: README.md [hunk 3]
```diff
@@ -139,6 +139,18 @@ runtime didn't start up properly (check your node process for instance), maybe
...
+typically Node. 
+
+There are two ways to fix it:
+
+1. Instruct kaocha-cljs to use a browser REPL.
+2. Prepare a dom for Node environment. For example, `(js/require "global-jsdom")`
```
Comment (alysbrooks) [inline README.md:151]:
```suggestion
2. Prepare a DOM for Node environment. For example, `(js/require "global-jsdom")` 
```

**Review by alysbrooks — CHANGES_REQUESTED** [review]
Thanks for adding this! I Just fixed some minor issues.

It might be nice to go into the tradeoffs between a browser REPL versus including a DOM. I'm not sure whether there are any downsides to `global-jsdom`, but I suspect it has some differences from a browser DOM. The pro would be avoiding the unreliability of connecting to a browser, however.

**Comment by humorless** [comment]
> Thanks for adding this! I Just fixed some minor issues.
> 
> It might be nice to go into the tradeoffs between a browser REPL versus including a DOM. I'm not sure whether there are any downsides to `global-jsdom`, but I suspect it has some differences from a browser DOM. The pro would be avoiding the unreliability of connecting to a browser, however.

I got the idea/information of the `global-jsdom` thing by googling, and I found Arne's reply on ClojureVerse discussions. I think it would be nice to also add this method.

**Pull request merged**